rasa/nlu/training_data/message.py

```diff
@@ -114,6 +112,16 @@ def build(
             data[ENTITIES] = entities
         return cls(text, data, **kwargs)
 
+    @classmethod
+    def build_from_action(
+        cls,
+        action_text: Optional[Text] = "",
+        action_name: Optional[Text] = "",
+        **kwargs,
```

Comment:
types please

Reply:
and a docstring please

Reply:
The docstring should also describe the parameters please (see https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e)

Reply:
Done!